.github/workflows/hlwm.yml

```diff
@@ -0,0 +1,111 @@
+name: HLWM CI
+
+# on: [push, pull_request]
+on:
+  push:
+    paths-ignore:
+    - '.mergify.yml'
+    - 'share/**'
+
+jobs:
+  build-test-current:
+    # Name the Job
+    name: Build and test on current ubuntu
+    # Set the type of machine to run on
+    runs-on: ubuntu-latest
+    container: hlwm/ci:focal
+
+    steps:
+      # Checks out a copy of your repository on the ubuntu-latest machine
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - uses: actions/cache@v1
+        name: Cache ~/.ccache
+        with:
+          path: ~/.ccache
+          key: focal-ccache
+
+      - uses: actions/cache@v1
+        name: Cache .tox-cache
+        with:
+          path: .tox-cache
+          key: focal-tox
+
+      - name: Build
+        run: |
+          ci/build.py --build-dir=build --cmake --cxx=g++-9 --cc=gcc-9 --build-type=Debug --ccache=$HOME/.ccache --compile
+
+      - name: Test
+        run: |
+          ci/build.py --build-dir=build --run-tests
+
+      - name: Codecov report
+        continue-on-error: true
+        run: |
+          wget -O codecov-io.bash https://codecov.io/bash
```

Comment:
This doesn't seem to work anymore:

  wget -O codecov-io.bash https://codecov.io/bash
  bash codecov-io.bash -f coverage.info
  shell: sh -e {0}
/__w/_temp/030faa66-a8fb-4057-a380-e8962ace7133.sh: 1: wget: not found
Error: Process completed with exit code 127.

Reply:
It seems wget is missing in our docker image. The best fix I can think of is to add it to the docker image (later?) because I don't see any other way to download the script.

Reply:
Didn't we use curl in the past? Maybe that is installed in the Docker image already

Reply:
Looking at the Dockerfile, there is no indicator for me that curl is installed. In the past, that line was executed outside of docker, namely on the system provided by travis. I'm sure that the system provided by github actions also has wget or curl installed, but I don't know how to perform a single step of a job outside the `container` specified.

Reply:
True. Let's fix that afterwards.